src/Controller/ConfigDataObjectController.php

```diff
@@ -92,6 +92,7 @@ public function saveAction(Request $request): ?JsonResponse
             }
             $config->setConfiguration($dataDecoded);
 
+            // @phpstan-ignore-next-line isAllowed return can changed now
```

Comment:
can you explain, why this line is necessary? `isAllowed` should return a bool value, shouldn't it?

Reply:
`$config->isAllowed('update')` is called on line 90, phpstan says that it is not necessary to double-check. But it can be changed because of `$config->setConfiguration($dataDecoded);`, right?